### DISCUSSION TITLE
CLI option to load specific configuration file on startup

### DISCUSSION THREAD

**Discussion by RobSpectre** [streamdeck-linux-gui/streamdeck-linux-gui#63] [Ideas / Feature Requests]
As a user, I'd like to specify the configuration file I would like to use on startup.

e.g. `streamdeck -n --config config.json`

**Comment by RemcoSchrijver** [comment]
Would be possible by overriding the STATE_FILE variable, this should do the trick. 

https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/blob/1db27737fa136bd27576718180c6053d84d619cc/streamdeck_ui/config.py#L11C9-L11C9

**Reply by RobSpectre** [reply]
Yup - this'll do!

Thanks for the quick reply on a holiday weekend.
